website/cue/reference/configuration.cue

```diff
@@ -252,6 +252,56 @@ configuration: {
 			}
 		}
 
+		telemetry: {
```

Comment:
I take it that this is not generated from the `configurable` trait? Maybe that doesn't work for global options?

Reply:
Yeah I'm pretty sure it only works for components.

Reply:
👀 but global options are marked with [configurable](https://github.com/vectordotdev/vector/blob/53f8bff371cdfa96770c03be38ae3c83a497043f/lib/vector-core/src/config/global_options.rs#L37)?

Reply:
cc/ @tobz to confirm. Maybe it just isn't handled by the Ruby script that turns the JSON schema into cue docs yet.

Reply:
There is no `reference/base` to include them in the cue file from..